src/graph/rome_models.cc

```diff
@@ -886,6 +898,7 @@ static struct rcclRomeModel romeTopoModels[] = {
   rome_model_81, /* 40 */
   rome_model_84, /* 41 */
   rome_model_85, /* 42 */
+  rome_model_87, /* 43 */
```

Comment:
Just curious, why no model 86?

Reply:
model 86 is used to test rail solution with odd number of gfx940 nodes.